lib/channels/src/index.ts

```diff
@@ -0,0 +1,152 @@
+export type ChannelHandler<TEventArgs = any> = (event: ChannelEvent<TEventArgs>) => void;
+
+export interface ChannelTransport<TEventArgs = any> {
+  send(event: ChannelEvent<TEventArgs>): void;
+  setHandler(handler: ChannelHandler<TEventArgs>): void;
+}
+
+export interface ChannelEvent<TEventArgs = any> {
```

Comment:
Continuing on above's comment: if it is necessary to have default values for generic types and this is the implementation of it, do we have to drill it down every interface and type?

Reply:
Default values are not necessary but prevent a breaking change. Without a default the consumer is forced to pass in an interface. However, as mentioned above I'm going to remove them